CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ option(FILAMENT_USE_EXTERNAL_GLES3 "Experimental: Compile Filament against OpenG
 
 option(FILAMENT_USE_SWIFTSHADER "Compile Filament against SwiftShader" OFF)
 
+option(FILAMENT_USE_MULTIVIEW_STEREO "Compile Filament against multiview extension for stereo rendering; This disables instanced stereo" OFF)
```

Comment:
Let's clarify that this only affects which stereo mode our samples use (i.e., gltf_viewer). I think this should go in our samples CMakeLists.txt (samples/CMakeLists.txt).

Can we also make this a string instead of a bool? Maybe `FILAMENT_SAMPLES_STEREO_TYPE` and let it be either "instanced" or "multiview".